plugin/github-link.vim

```diff
@@ -43,6 +43,13 @@ function! s:execute_with_ref(ref, startline, endline)
 
     " https://github.com/OWNER/REPO/blob/BRANCH/PATH/FROM/ROOT#LN-LM
     let s:link = s:repo . "/blob/" . a:ref . "/" . s:path_from_root
+
+    " Check for doc extensions and add plain query parameter, because otherwise
+    " GitHub ignores the line highlight
+    if s:link =~? '\v.*\.(md|rst|markdown|mdown|mkdn|md|textile|rdoc|org|creole|mediawiki|wiki|rst|asciidoc|adoc|asc|pod)$' && s:remote =~ '.*github.*'
```

Comment:
`\v` stands for "don't use vim's weird regex"

The `?` in `=~?` stands for "case insensitive regex" (GitHub rendering of
doc files is also case insensitive)